youtube_dl/extractor/rai.py

```diff
@@ -455,3 +455,29 @@ def _real_extract(self, url):
         info.update(relinker_info)
 
         return info
+
+
+class RaiPlaylistIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?raiplay\.it/programmi/(?P<id>[^/]+)'
+    _TESTS = [{
+        'url': 'http://www.raiplay.it/programmi/nondirloalmiocapo/',
+        'info_dict': {
+            'id': 'nondirloalmiocapo',
+            'title': 'Non dirlo al mio capo',
+        },
+        'playlist_mincount': 12,
+    }]
+
+    def _real_extract(self, url):
+        playlist_id = self._match_id(url)
+        webpage = self._download_webpage(url, playlist_id)
+        title = self._html_search_meta('programma', webpage, default=None)
+        entries = [
+            self.url_result(
+                urljoin(url, video_url),
+                RaiPlayIE.ie_key())
+            for video_url in re.findall(
+                r'<a class="video" href="([^"]+)"',
```

Comment:
As said it should be more relaxed allowing both kinds of quotes, attributes in between and reversed order of attributes.